coderd/prometheusmetrics/aggregator_test.go

```diff
@@ -44,13 +44,35 @@ func TestUpdateMetrics_MetricsDoNotExpire(t *testing.T) {
 
 	given2 := []agentsdk.AgentMetric{
 		{Name: "b_counter_two", Type: agentsdk.AgentMetricTypeCounter, Value: 4},
+		{Name: "c_gauge_three", Type: agentsdk.AgentMetricTypeGauge, Value: 5},
+		{Name: "c_gauge_three", Type: agentsdk.AgentMetricTypeGauge, Value: 2, Labels: []agentsdk.AgentMetricLabel{
+			// Metrics are sorted alphabetically, and we have to prepend "z", so common labels (agent name, username, workspace name) go first.
```

Comment:
this feels fragile.

Your tests could build string maps of the expected and actual labels and then `require.Equal` on the maps.

Reply:
Fixed